tmt/steps/prepare/artifact/providers/repository.py

```diff
@@ -134,6 +140,24 @@ def fetch_contents(
 
         return []
 
+    def contribute_to_shared_repo(
+        self,
+        guest: Guest,
+        download_path: Path,
+        shared_repo_dir: Path,
+        exclude_patterns: Optional[list[Pattern[str]]] = None,
+    ) -> None:
+        # Repository discovery happens after installation by PrepareArtifact.
+        # We don't call fetch_contents here as it requires the repository to be installed first.
+        pass
+
+    def get_repositories(self) -> list[Repository]:
+        self.logger.info(
+            f"Providing repository '{self.repository.name}' for installation "
+            f"(repo IDs: {', '.join(self.repository.repo_ids)})"
+        )
```

Comment:
Isn't this log already replicated in the `create_repository`? Drop the `repo IDs` part in one of them